images/server/install-sambacc.sh

```diff
@@ -34,7 +34,7 @@ case $action in
         container_json_file="/usr/local/share/sambacc/examples/minimal.json"
     ;;
     install-rpm)
-        dnf install -y "${rpmfiles[0]}"
+        dnf install  -y "${rpmfiles[0]}"
```

Comment:
Extra space before `-y`. This change can be taken out.

Reply:
```suggestion
        dnf install -y "${rpmfiles[0]}"
```